class-argument-converter/pom.xml

```diff
@@ -40,6 +40,12 @@
       <artifactId>junit-jupiter-params</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
Why do all of these poms need this dependency all of a sudden?

Reply:
> Why do all of these poms need this dependency all of a sudden?

I updated surefire-plugin and removed junit-platform-surefire-provider.

See:
https://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven-engines-configure
Starting with version 2.22.0, Maven Surefire and Maven Failsafe provide native support for executing tests on the JUnit Platform. The pom.xml file in the junit5-jupiter-starter-maven project demonstrates how to use the Maven Surefire plugin and can serve as a starting point for configuring your Maven build.

In order to have Maven Surefire or Maven Failsafe run any tests at all, at least one TestEngine implementation must be added to the test classpath.